src/main/java/org/bytedeco/javacpp/tools/Generator.java

```diff
@@ -2401,16 +2401,28 @@ void parametersBefore(MethodInformation methodInfo) {
 
     String returnBefore(MethodInformation methodInfo) {
         String returnPrefix = "";
+        String[] typeName =
+            methodInfo.allocator || methodInfo.arrayAllocator ? cppTypeName(methodInfo.cls) :
+                (methodInfo.returnRaw ? new String[]{""}
+                    : cppCastTypeName(methodInfo.returnType, methodInfo.annotations));
```

Comment:
Is this doing anything different from the existing code in master?

Reply:
You mean the move of typeName definition at the beginning of the method ? I guess I did that for consistency with returnAfter. But I can rechange for something closer to master.

Reply:
Well, if it's not supposed to change anything, let's not "fix it" if it's _ain't_ broken. The original code looks fine to me.

Reply:
Done